WalletWasabi.Fluent/CrashReport/CrashReporter.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Diagnostics;
+using WalletWasabi.Logging;
+using WalletWasabi.Microservices;
+using WalletWasabi.Models;
+
+namespace WalletWasabi.Fluent.CrashReport
+{
+	public static class CrashReporter
+	{
+		public static bool TryInvokeIfRequired(Exception exceptionToReport)
```

Comment:
The `Try*` names are good for functions where it is expected to return true of false based on some data structure content. However, your function should succeed always and if there is a bug then it may fail. So I would name it "Invoke" / "InvokeReporter" or something.

Reply:
ACK